src/attach.ts

```diff
@@ -1,6 +1,6 @@
 import WebSocket = require('isomorphic-ws');
 import querystring = require('querystring');
```

Comment:
Couldn't we use `import querystring = require('node:querystring');` here is well?

Reply:
Updated. Thanks!